dev/lint-r.R

```diff
@@ -20,14 +20,13 @@ SPARK_ROOT_DIR <- as.character(argv[1])
 LOCAL_LIB_LOC <- file.path(SPARK_ROOT_DIR, "R", "lib")
 
 # Checks if SparkR is installed in a local directory.
-if (! library(SparkR, lib.loc = LOCAL_LIB_LOC, logical.return = TRUE)) {
+if (!requireNamespace("SparkR", lib.loc = LOCAL_LIB_LOC)) {
   stop("You should install SparkR in a local directory with `R/install-dev.sh`.")
 }
 
-# Installs lintr from Github in a local directory.
-# NOTE: The CRAN's version is too old to adapt to our rules.
-if ("lintr" %in% row.names(installed.packages()) == FALSE) {
-  devtools::install_github("jimhester/lintr@v2.0.0")
+# Installs lintr if needed
+if (!requireNamespace("lintr")) {
+  install.packages('lintr')
```

Comment:
Looks like there was sort of a reason for installing a particular version: https://github.com/apache/spark/pull/26564#discussion_r347198119 But the logic seems to be "use the new 2.0.0 version". This would always pick up the latest. I'm guessing that's OK, @dongjoon-hyun ?

Reply:
the comment said the CRAN version wasn't recent enough so I assumed `>= 2.0.0` would be fine. & we're passing so that's a good sign.

Anyway, yes, would be good to confirm on this.

Reply:
Ya. It looks okay since this seems to install lintr 2.0.1 as of today.